corehq/messaging/scheduling/models/content.py

```diff
@@ -712,6 +712,12 @@ def send(self, recipient, logged_event, phone_entry=None):
         send_message_to_verified_number(connect_number, message, logged_subevent=logged_subevent)
 
 class ConnectMessageSurveyContent(SurveyContent):
+    def get_critical_section(self, recipient):
```

Comment:
I'd pull this up into `Content` rather than having the same implementation in two subclasses.